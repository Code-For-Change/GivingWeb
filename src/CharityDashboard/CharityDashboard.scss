$height: calc(100vh - (58px));

.charity-dashboard {
	width: 100%;
	display: flex;
	justify-content: center;
	background: #f5f5f5;

	.main-section {
		min-height: $height;
		width:100vw ;
		max-width: 1300px;
		display: flex;
		overflow: hidden;
		margin-top: 25px;
		margin-bottom: 25px;
		background: #fff;
		// border-radius: 35px;
    box-shadow: 0px .01px 1px 0.1px #888888;
	}
}
