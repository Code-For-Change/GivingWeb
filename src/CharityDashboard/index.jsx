import React from 'react'
import { Route } from 'react-router'

import css from './CharityDashboard.scss'

import Sidebar from './components/Sidebar'
import Content from './components/Content'
import CampaignsLive from './components/CampaignsLive'
import CampaignsDraft from './components/CampaignsDraft'
import CampaignsReview from './components/CampaignsReview'
import CampaignsSuccess from './components/CampaignsSuccess'


class CharityDashboard extends React.Component {
	render() {
		return (
			<div className="charity-dashboard">
				<div className="main-section">
					<Sidebar />
					<Content />
				</div>
			</div>
		)
	}
}

			

export default CharityDashboard