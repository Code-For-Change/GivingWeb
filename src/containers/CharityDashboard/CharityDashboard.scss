$height: calc(100vh - (58px));

.charity-dashboard {
	width: 100%;
	display: flex;
	justify-content: center;

	.main-section {
		min-height: $height;
		width: 90%;
		display: flex;

		.dashboard-current-page {
			height: $height;
			width: 80%;
			background-color: red;
			
			.breadcrumbs {
				padding: 10px;
				width: 100%;
				height: 60px;
				background-color: green;
				display: flex;
				align-items: center;
			}
		}

	}
}