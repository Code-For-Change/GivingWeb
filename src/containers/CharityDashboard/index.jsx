import React from 'react'

import css from './CharityDashboard.scss'

import CharitySidebarNav from '../../components/CharitySidebarNav'

class CharityDashboard extends React.Component {
	render() {
		return (
			<div className="charity-dashboard">
				<div className="main-section">
					<CharitySidebarNav />
					<div className="dashboard-current-page">
						<div className="breadcrumbs"></div>
					</div>
				</div>

			</div>
		)
	}
}


export default CharityDashboard