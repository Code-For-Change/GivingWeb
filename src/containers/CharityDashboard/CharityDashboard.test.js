import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'

import CharityDashboard from './index'

describe('CharityDashboard', () => {

	it('should render correctly', () => {
    const wrapper = shallow(<CharityDashboard />)
    expect(toJson(wrapper)).toMatchSnapshot()
	})

});