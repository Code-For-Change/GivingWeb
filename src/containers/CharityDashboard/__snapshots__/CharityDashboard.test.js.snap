// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharityDashboard should render correctly 1`] = `
<div
  className="charity-dashboard"
>
  <div
    className="main-section"
  >
    <CharitySidebarNav />
    <div
      className="dashboard-current-page"
    >
      <div
        className="breadcrumbs"
      />
    </div>
  </div>
</div>
`;
